controllers/scenario/predicates.go

```diff
@@ -21,15 +21,15 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 )
 
-// IntegrationPipelineRunStartedPredicate returns a predicate which filters out
-// only created integration scenarios
+// IntegrationScenarioCreatedPredicate returns a predicate which filters out
```

Comment:
I feel that the functionname is little bit misleading, can we consider something like: 
_IntegrationScenarioCreatedAndDeletedPredicate_ ?

Reply:
Tbh I was thinking of removing the predicate altogether as we're responding to basically all types of events now.

Reply:
Ack, updated

Reply:
Do we really want to react to generic events?

Reply:
I'm fine with reacting to all events for this owned resource and making sure it's validated tbh